hack/tilt/kubetail.yaml

```diff
@@ -1,5 +1,115 @@
 kind: ConfigMap
 apiVersion: v1
+metadata:
+  name: kubetail-dashboard-config
```

Comment:
To be consistent with the other resources in the manifest let's use "kubetail-dashboard"